patch-upstream.sh

```diff
@@ -24,7 +24,7 @@ CURRENT_DIR=$PWD
 cd opentelemetry-lambda/collector
 
 # patch otel version on collector/go.mod
-PATCH_OTEL_VERSION="../../OTEL_Version.patch"
+PATCH_OTEL_VERSION="../../patches/OTEL_Version.patch"
```

Comment:
Can we instead change the location of the patch file instead of changing this?

Reply:
Agreed, I will change it.